propka/run.py

```diff
@@ -36,34 +36,69 @@ def main(optargs=None):
             my_molecule.write_propka()
 
 
-def single(pdbfile, optargs=None):
-    """Run a single PROPKA calculation using *pdbfile* as input.
+def single(filename: str, optargs: list = None, stream=None,
+           write_pka: bool = True):
+    """Run a single PROPKA calculation using ``filename`` as input.
 
-    Commandline options can be passed as a **list** in *optargs*.
+    Args:
```

Comment:
Document that only a single `filename` will be processed and that you effectively ignore `-f` (at least that's my reading of the discussion below).

Reply:
`Notes` section was added with this in 46de602